### PR TITLE
Package coq-hoare-tut.v8.11.1

### PR DESCRIPTION
**Pull request by k4rtik**
### `coq-hoare-tut.v8.11.1`
A Tutorial on Reflecting in Coq the generation of Hoare proof obligations
Hoare logics are "program logics" suitable for reasoning about
imperative programs.
This work is both an introduction to Hoare logic and a demo
illustrating Coq nice features. It formalizes the generation of PO
(proof obligations) in a Hoare logic for a very basic imperative
programming language. It proves the soundness and the completeness of
the PO generation both in partial and total correctness. At last, it
examplifies on a very simple example (a GCD computation) how the PO
generation can simplify concrete proofs. Coq is indeed able to compute
PO on concrete programs: we say here that the generation of proof
obligations is reflected in Coq. Technically, the PO generation is
here performed through Dijkstra's weakest-precondition calculus.

---
* Homepage: https://github.com/coq-community/hoare-tut
* Source repo: git+https://github.com/coq-community/hoare-tut.git
* Bug tracker: https://github.com/coq-community/hoare-tut/issues

---
:camel: Pull-request generated by opam-publish v2.0.2

### CODE REVIEW

File: released/packages/coq-hoare-tut/coq-hoare-tut.v8.11.1/opam [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  "keyword:imperative program"
+  "keyword:weakest precondition"
+  "keyword:reflection"
+  "logpath:HoareTut"
+]
+authors: [
+  "Sylvain Boulmé"
+]
+url {
+  src: "https://github.com/coq-community/hoare-tut/archive/v8.11.1.tar.gz"
```
Comment (gares) [inline released/packages/coq-hoare-tut/coq-hoare-tut.v8.11.1/opam:44]: I believe the dependency on Coq should mention a version. Given this URL, I imagine it is >= 8.11

Comment (k4rtik) [reply released/packages/coq-hoare-tut/coq-hoare-tut.v8.11.1/opam:44]: Thanks for catching this. I am asking this question at [Zulip chat](https://coq.zulipchat.com/#narrow/stream/237663-coq-community-devs.20.26.20users/topic/author.20in.20opam.20packages/near/199862776) as well. But do you think mentioning something such as `>=8.10` is going to be enough?

**Comment by k4rtik** [comment]
For maintainers, I would like to confirm if the file path should be:
- `released/packages/coq-hoare-tut/coq-hoare-tut.v8.11.1/opam`, or
- `released/packages/coq-hoare-tut/coq-hoare-tut.8.11.1/opam`

The `opam-publish` tool I used to create this PR created the former and in the second commit I realized the issue and changed it to the latter.

**Comment by palmskog** [comment]
@k4rtik this looks good now, but maybe you can also do similar changes (to Coq version restrictions) to the recently submitted package for Semantics (i.e., for `coq-semantics.8.11.1`)?

**Pull request merged**